custom_components/kia_uvo/KiaUvoApiCA.py

```diff
@@ -124,7 +124,12 @@ def get_cached_vehicle_status(self, token: Token):
         vehicle_status["odometer"] = {}
         vehicle_status["odometer"]["unit"]= response["currentOdometerUnit"]
         vehicle_status["odometer"]["value"]= response["currentOdometer"]
-        vehicle_status["vehicleLocation"] = self.get_location(token)
+        
+        vehicle_status["vehicleLocation"] = {}
+        try:
```

Comment:
i personally would have try/catch block inside get_location function and return empty dictionary `{}` for any error. So all callers would benefit it. 

What happens if we don't have lat and lon items in vehicle_location? Any further error? Maybe we can create these items on empty response.

Reply:
I have moved the try catch. I don't want to return invalid location as the last known location may still be correct.  Thoughts?